### DISCUSSION TITLE
GitHub Packages Assets are not download links

### DISCUSSION THREAD

**Discussion by BlackSwan2019** [community/community#41929] [Actions]
### Select Topic Area

Bug

### Body

The download links for package assets is just static text, like [here](https://github.com/Codertocat/hello-world-npm/pkgs/npm/hello-world-npm/31741573).

Shouldn't they be blue and clickable to get the asset?

![disabledAssetsLink](https://user-images.githubusercontent.com/21174143/208162262-2f67466b-3b9e-4ca8-97c9-6a37cb05f465.png)

**Comment by marisks** [comment]
The previous version of GitHub packages had downloadable assets. Now it makes it quite hard to publish packages in other feeds - I had to setup a project locally, install the package, and then upload it to the feed. Previously, I could just download it from GitHub and then upload it to NuGet.org or any other feed my package needs to be.

**Comment by felipenmoura** [comment]
yep...that's problematic!
There are no advantages in not having the link there. Why would they do that?
I too had to download, install, build and zip the result, instead of ... you know, clicking a link.

**Comment by trickyearlobe** [comment]
I have a corporate customer which restricts internet access (quite common) and uses PAC files to configure proxy support on the browser.

They can't reach out for artifact repos like RubyGems, YUM/APT repos, Maven etc, NPM etc, but they can open a browser to download an artifact directly.

Not allowing "click to download" means they can't get my stuff without shady file transfer solutions or USB sticks.

**Reply by trickyearlobe** [reply]
Also, why no YUM/APT support?
